### ISSUE TITLE
Patch Classification notebook, key error with data loaders

### ISSUE THREAD

**Issue by Joe-Fin** [TissueImageAnalytics/tiatoolbox#339]
* TIA Toolbox version:
* Python version: 3
* Operating System: Colab

### Description

Tried to utilise resnet18-kather100k pretrained model for transfer learning (prediction), the issue seems to be the same with other pretrained models also, but this may need a double check. Saad also had the same issue after answering for verification
The error is a missing and unexpected keys when trying to initialise the state dict.

I believe I know what is causing the issue which is that in either the unweighted model, or the resnet18-kather100k model itself, there are two mispelt keys 'classifer.weight' and 'classifer.bias' these should both be 'classifier'. 

### What I Did
Ran the line:
predictor = PatchPredictor(pretrained_model='resenet18-kather100k', batch_size=32)
```
RuntimeError: Error(s) in loading state_dict for CNNmodel:
Missing key(s) in state_dict: 'classifer.weight', 'classifer.bias'.
Unexpected key(s) in state_dict: 'classifier.weight', 'classifier.bias'.

```

**Comment by shaneahmed** [comment]
The error is due to a typo which was recently fixed on the develop branch https://github.com/TissueImageAnalytics/tiatoolbox/commit/862d82f95096975ac571e3d2d2bbbfbbfd040765#diff-a84858f695910c0026d8a467831bf4f1e12fe67d6f15365d6143b257f7445692R98

This issue will be resolved in the next release when changes from develop are merged with the main branch. 

For now, you can try "pip install --upgrade --ignore-installed git+https://github.com/TissueImageAnalytics/tiatoolbox.git@develop" in your environment to run the notebook.

**Comment by shaneahmed** [comment]
Fixed in release 1.1.0 https://github.com/TissueImageAnalytics/tiatoolbox/releases/tag/v1.1.0
